api/s2n.h

```diff
@@ -333,6 +333,39 @@ S2N_API
 extern int s2n_connection_set_read_fd(struct s2n_connection *conn, int readfd);
 S2N_API
 extern int s2n_connection_set_write_fd(struct s2n_connection *conn, int writefd);
+
+/**
+  * Gets the assigned fd for the read channel of an s2n connection.
+  *
+  * In case the s2n connection was set to managed io on the read channel
+  * via s2n_connecion_set_fd or s2n_connection_set_read_fd.
+  *
+  * @param conn A pointer to the s2n connection
+  * @param readfd pointer to place the used fd. in case the connection was not assigned
+  * a read side managed fd by either calling s2n_connecion_set_fd or s2n_connection_set_read_fd
+  * will set the value of -1 to the resulting fd.
```

Comment:
```suggestion
  * @param readfd pointer to store the file descriptor. If the connection was not assigned
  * a read fd by either calling s2n_connecion_set_fd or s2n_connection_set_read_fd, `readfd` will be -1.
```

Reply:
not relevant now that I removed the -1 logic